src/Components/Icon/IconExamples.md

```diff
@@ -4,113 +4,114 @@
+<div className="__icon"><Icon name="schedule" accessibilityLabel="schedule" /><div>schedule</div></div>
+<div className="__icon"><Icon name="s-check" accessibilityLabel="s-check" /><div>s-check</div></div>
+<div className="__icon"><Icon name="remove" accessibilityLabel="remove" /><div>remove</div></div>
+<div className="__icon"><Icon name="search" accessibilityLabel="search" /><div>search</div></div>
+<div className="__icon"><Icon name="settings" accessibilityLabel="settings" /><div>settings</div></div>
+<div className="__icon"><Icon name="shopping-bag" accessibilityLabel="shopping-bag" /><div>shopping-bag</div></div>
+<div className="__icon"><Icon name="settings-gear" accessibilityLabel="settings-gear" /><div>settings-gear</div></div>
+<div className="__icon"><Icon name="shop" accessibilityLabel="shop" /><div>shop</div></div>
+<div className="__icon"><Icon name="stop" accessibilityLabel="stop" /><div>stop</div></div>
+<div className="__icon"><Icon name="source" accessibilityLabel="source" /><div>source</div></div>
+<div className="__icon"><Icon name="sound-on" accessibilityLabel="sound-on" /><div>sound-on</div></div>
+<div className="__icon"><Icon name="sound-off" accessibilityLabel="sound-off" /><div>sound-off</div></div>
+<div className="__icon"><Icon name="stack" accessibilityLabel="stack" /><div>stack</div></div>
+<div className="__icon"><Icon name="tag" accessibilityLabel="tag" /><div>tag</div></div>
+<div className="__icon"><Icon name="ticket" accessibilityLabel="ticket" /><div>ticket</div></div>
+<div className="__icon"><Icon name="users" accessibilityLabel="users" /><div>users</div></div>
+<div className="__icon"><Icon name="sports" accessibilityLabel="sports" /><div>sports</div></div>
+<div className="__icon"><Icon name="video" accessibilityLabel="video" /><div>video</div></div>
+<div className="__icon"><Icon name="tv-channel" accessibilityLabel="tv-channel" /><div>tv-channel</div></div>
+<div className="__icon"><Icon name="spinner" accessibilityLabel="spinner" /><div>spinner</div></div>
+<div className="__icon"><Icon name="visible" accessibilityLabel="visible" /><div>visible</div></div>
+<div className="__icon"><Icon name="warning-triangle" accessibilityLabel="warning-triangle" /><div>warning-triangle</div></div>
+<div className="__icon"><Icon name="warning-circle" accessibilityLabel="warning-circle" /><div>warning-circle</div></div>
+<div className="__icon"><Icon name="vod" accessibilityLabel="vod" /><div>vod</div></div>
+<div className="__icon"><Icon name="warning-circle-outline" accessibilityLabel="warning-circle-outline" /><div>warning-circle-outline</div></div>
+<div className="__icon"><Icon name="workflow" accessibilityLabel="workflow" /><div>workflow</div></div>
 <div className="__icon"><Icon name="world" accessibilityLabel="world" /><div>world</div></div>
-  </div>
+<div className="__icon"><Icon name="delete" accessibilityLabel="delete" /><div>delete</div></div>
```

Comment:
is this file autogenerated? If not, can this be alphabetized?

Reply:
it's autogenerated.